pom.xml

```diff
@@ -50,7 +50,7 @@
         <dep.packaging.version>${dep.airlift.version}</dep.packaging.version>
         <dep.slice.version>0.27</dep.slice.version>
         <dep.aws-sdk.version>1.11.30</dep.aws-sdk.version>
-        <dep.tempto.version>1.22</dep.tempto.version>
+        <dep.tempto.version>1.23</dep.tempto.version>
```

Comment:
I already updated tempto to 1.23 on prestodb/master. This will be removed while merging.